src/libraries/System.IO/tests/BufferedStream/BufferedStreamTests.cs

```diff
@@ -369,4 +419,54 @@ internal sealed class ThrowsExceptionFromAsyncOperationsStream : MemoryStream
         public override Task FlushAsync(CancellationToken cancellationToken) =>
             throw new InvalidOperationException("Exception from FlushAsync");
     }
+
+    internal sealed class WriteOnlyStream : Stream
+    {
+        private long _pos;
+
+        public override void Flush()
+        {
+        }
+
+        public override int Read(byte[] buffer, int offset, int count)
+        {
+            throw new NotSupportedException();
+        }
+
+        public override long Seek(long offset, SeekOrigin origin)
+        {
+            throw new NotSupportedException();
+        }
+
+        public override void SetLength(long value)
+        {
+            throw new NotSupportedException();
+        }
+
+        public override void Write(byte[] buffer, int offset, int count)
+        {
+            _pos += (count - offset);
+        }
+
+        public override void Write(ReadOnlySpan<byte> buffer)
+        {
+            _pos += buffer.Length;
+        }
+
+        public override bool CanRead => false;
+        public override bool CanSeek => false;
+        public override bool CanWrite => true;
+        public override long Length => _pos;
+
+        public override long Position
+        {
+            get => throw new NotSupportedException();
+            set => throw new NotSupportedException();
+        }
+
+        public long GetPosition()
+        {
+            return _pos;
+        }
+    }
```

Comment:
Rather than this method,  the value can be returned from get_Position

Reply:
Fixed.